Microsoft.Toolkit.Uwp.UI.Controls/UniformGrid/UniformGrid.cs

```diff
@@ -34,25 +34,21 @@ public partial class UniformGrid : Grid
         // Internal list we use to keep track of items that we don't have space to layout.
         private List<UIElement> _overflow = new List<UIElement>();
 
-        /// <summary>
-        /// Measure the controls before layout.
-        /// </summary>
-        /// <param name="availableSize">Size available from parent.</param>
-        /// <returns>Desired Size</returns>
+        /// <inheritdoc/>
         protected override Size MeasureOverride(Size availableSize)
         {
             // Get all Visible FrameworkElement Children
-            var visible = Children.Where(item => item.Visibility != Visibility.Collapsed && item is FrameworkElement).Select(item => item as FrameworkElement);
+            var visible = Children.Where(item => item.Visibility != Visibility.Collapsed && item is FrameworkElement).Select(item => item as FrameworkElement).ToArray();
 
-            var dim = GetDimensions(ref visible, Rows, Columns, FirstColumn);
+            (int Rows, int Columns) dim = GetDimensions(visible, Rows, Columns, FirstColumn);
```

Comment:
Or you can use Tuple deconstruction `var (rows, columns) = …;`.

https://docs.microsoft.com/fr-fr/dotnet/csharp/deconstruct